x/attribute/client/cli/cli_test.go

```diff
@@ -591,6 +593,83 @@ func (s *IntegrationTestSuite) TestAttributeTxCommands() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestAttributeTxCommands_WithExpiration() {
+
+	testCases := []struct {
+		name            string
+		cmd             *cobra.Command
+		args            []string
+		expectErr       bool
```

Comment:
An `expectErr bool` is almost never worthwhile. If you're expecting an error, you should testing the error message for specific content. Otherwise, things can be going drastically wrong in unexpected ways, but the test will pass because it doesn't realize the error received has nothing to do with the test.